alws/routers/builds.py

```diff
@@ -35,14 +36,18 @@
 @router.post('/', response_model=build_schema.BuildCreateResponse)
 async def create_build(
             build: build_schema.BuildCreate,
-            user: dict = Depends(JWTBearer()),
+            user: models.User = Depends(get_current_user),
             db: database.Session = Depends(get_db)
         ):
-    return await build_crud.create_build(
-        db,
-        build,
-        user['identity']['user_id']
-    )
+    # `get_current_user returns tuple of user record and token,
+    # so to address user record we need to pick first item in `user_data`
+    try:
+        return await build_crud.create_build(db, build, user.id)
```

Comment:
in comment above you mention that `get_current_user` returns tuple, but you use `id` attribute here

Reply:
Yeah, didn't remove the comment, it's actually returning a User object